.gitignore

```diff
@@ -13,3 +13,6 @@ public
 resources
 node_modules
 generated
+install
```

Comment:
Add a comment: # Install generated files

Reply:
Based on istio/istio, does this need to be install/** if we are only adding one line?

Reply:
Done.